cmake/autodetect_system_libs.cmake

```diff
@@ -0,0 +1,7 @@
+if (NOT DEFINED GINKGO_BUILD_HWLOC)
+    find_package(HWLOC 2.1)
+endif()
+
+if (NOT DEFINED GINKGO_BUILD_PAPI_SDE)
+    find_package(PAPI 7.0.1.0 COMPONENTS sde)
+endif()
```

Comment:
The not-found message is not an error message. Is it still the case?
From previous comment, `We do not need to use "QUIET" because we ship the Find*.cmake. `

Reply:
Yes, here it's not an error message when it's not found, there's only one status line about this to tell that the detection succeeded/failed.